src/Symfony/Bundle/FrameworkBundle/Resources/config/http_client.php

```diff
@@ -49,13 +50,20 @@
 
         ->alias(ClientInterface::class, 'psr18.http_client')
 
-        ->set(\Http\Client\HttpClient::class, HttplugClient::class)
+        ->set(HttpAsyncClient::class, HttplugClient::class)
             ->args([
                 service('http_client'),
                 service(ResponseFactoryInterface::class)->ignoreOnInvalid(),
                 service(StreamFactoryInterface::class)->ignoreOnInvalid(),
             ])
 
+        ->alias(\Http\Client\HttpClient::class, HttpAsyncClient::class)
+            ->deprecate(
+                'symfony/http-client',
+                '6.3',
+                'The "%alias_id%" service is deprecated, use the "'.HttpAsyncClient::class.'" service for method "::sendAsyncRequest" and "'.ClientInterface::class.'" for "::sendRequest" instead',
```

Comment:
`'The "%alias_id%" service is deprecated, use "'.ClientInterface::class.'" instead',` (the deprecation doesn't need to tell about something different than what is used at the deprecation point)

Reply:
True, but this was the only way to receive the `HttplugClient` with existing services which then again provides both.
Looking at the interface level it is surely correct to not mention it.
I prefer longer deprecations with more help to catch some additional lost souls over the very short messages.